### ISSUE TITLE
本地搭建出现问题

### ISSUE THREAD

**Issue by qixidao** [harry0703/AudioNotes#2]
`conda create -n AudioNotes python=3.10 -y
conda activate AudioNotes
git clone https://github.com/harry0703/AudioNotes.git
cd AudioNotes
pip install -r requirements.txt`

requirements.txt 这个文件代码里没提供啊？我卡在这步了

**Comment by zhaoxbmsdn** [comment]
你可以**手动安装依赖**，根据提示逐步完善；
这是一份我导出的依赖，你可以用来试试：
`aiofiles==23.2.1
aiohttp==3.9.5
aiosignal==1.3.1
aliyun-python-sdk-core==2.15.1
aliyun-python-sdk-kms==2.16.3
annotated-types==0.7.0
antlr4-python3-runtime==4.9.3
anyio==3.7.1
async-timeout==4.0.3
asyncer==0.0.2
asyncio==3.4.3
asyncpg==0.29.0
attrs==23.2.0
audioread==3.0.1
bidict==0.23.1
certifi==2024.7.4
cffi==1.16.0
chainlit==1.1.306
charset-normalizer==3.3.2
chevron==0.14.0
click==8.1.7
colorama==0.4.6
crcmod==1.7
cryptography==43.0.0
dataclasses-json==0.5.14
decorator==5.1.1
Deprecated==1.2.14
distro==1.9.0
editdistance==0.8.1
einops==0.8.0
exceptiongroup==1.2.2
fastapi==0.110.3
filelock==3.15.4
filetype==1.2.0
frozenlist==1.4.1
fsspec==2024.6.1
funasr==1.1.2
googleapis-common-protos==1.63.2
greenlet==3.0.3
grpcio==1.65.1
h11==0.14.0
httpcore==1.0.5
httpx==0.27.0
hydra-core==1.3.2
idna==3.7
importlib_metadata==7.1.0
intel-openmp==2021.4.0
jaconv==0.3.4
jamo==0.4.1
jieba==0.42.1
Jinja2==3.1.4
jmespath==0.10.0
joblib==1.4.2
kaldiio==2.18.0
Lazify==0.4.0
lazy_loader==0.4
librosa==0.10.2.post1
literalai==0.0.607
llvmlite==0.43.0
loguru==0.7.2
MarkupSafe==2.1.5
marshmallow==3.21.3
mkl==2021.4.0
mpmath==1.3.0
msgpack==1.0.8
multidict==6.0.5
mypy-extensions==1.0.0
nest-asyncio==1.6.0
networkx==3.3
numba==0.60.0
numpy==1.26.4
omegaconf==2.3.0
openai==1.36.1
opentelemetry-api==1.25.0
opentelemetry-exporter-otlp==1.25.0
opentelemetry-exporter-otlp-proto-common==1.25.0
opentelemetry-exporter-otlp-proto-grpc==1.25.0
opentelemetry-exporter-otlp-proto-http==1.25.0
opentelemetry-instrumentation==0.46b0
opentelemetry-proto==1.25.0
opentelemetry-sdk==1.25.0
opentelemetry-semantic-conventions==0.46b0
oss2==2.18.6
packaging==23.2
platformdirs==4.2.2
pooch==1.8.2
protobuf==4.25.3
psycopg2==2.9.9
pycparser==2.22
pycryptodome==3.20.0
pydantic==2.8.2
pydantic_core==2.20.1
PyJWT==2.8.0
pynndescent==0.5.13
python-dotenv==1.0.1
python-engineio==4.9.1
python-multipart==0.0.9
python-socketio==5.11.3
pytorch-wpe==0.0.1
PyYAML==6.0.1
requests==2.32.3
rotary-embedding-torch==0.6.4
scikit-learn==1.5.1
scipy==1.14.0
sentencepiece==0.2.0
simple-websocket==1.0.0
six==1.16.0
sniffio==1.3.1
soundfile==0.12.1
soxr==0.3.7
SQLAlchemy==2.0.31
starlette==0.37.2
sympy==1.13.1
syncer==2.0.3
tbb==2021.13.0
tensorboardX==2.6.2.2
threadpoolctl==3.5.0
tomli==2.0.1
torch==2.3.1
torch-complex==0.4.4
torchaudio==2.3.1
tqdm==4.66.4
typing==3.7.4.3
typing-inspect==0.9.0
typing_extensions==4.12.2
umap-learn==0.5.6
uptrace==1.24.0
urllib3==2.2.2
uuid==1.30
uvicorn==0.25.0
watchfiles==0.20.0
win32-setctime==1.1.0
wrapt==1.16.0
wsproto==1.2.0
yarl==1.9.4
zipp==3.19.2
`

**Comment by harry0703** [comment]
> `conda create -n AudioNotes python=3.10 -y conda activate AudioNotes git clone https://github.com/harry0703/AudioNotes.git cd AudioNotes pip install -r requirements.txt`
> 
> requirements.txt 这个文件代码里没提供啊？我卡在这步了

漏提交了😄，已经补上去了
